psi4/driver/procrouting/proc.py

```diff
@@ -3744,38 +3747,58 @@ def format_vector(label, data):
     else:
         raise ValidationError(f"Gauge {gauge} not recognised for ADC calculations.")
 
-    computed = {}
-    if any(prop in properties for prop in ("TRANSITION_DIPOLE", "OSCILLATOR_STRENGTH")):
-        data = state.transition_dipole_moment
-        computed["Transition dipole moment (in a.u.)"] = data
-        adc_wfn.set_variable(f"{name} transition dipoles", core.Matrix.from_array(data))
-
-    if "OSCILLATOR_STRENGTH" in properties:
-        if gauge == "velocity":
-            data = state.oscillator_strength_velocity.reshape(-1, 1)
-        else:
-            data = state.oscillator_strength.reshape(-1, 1)
-        computed[f"Oscillator strength ({gauge} gauge)"] = data
-        adc_wfn.set_variable(f"{name} oscillator strengths ({gauge_short})",
-                             core.Matrix.from_array(data))
-
-    if "ROTATIONAL_STRENGTH" in properties:
-        data = state.rotatory_strength.reshape(-1, 1)
-        computed["Rotational strength (velocity gauge)"] = data
-        adc_wfn.set_variable(f"{name} rotational strengths (VEL)",
-                             core.Matrix.from_array(data))
-
-    if "DIPOLE" in properties:
-        data = state.state_dipole_moment
-        computed["State dipole moment (in a.u.)"] = data
-        adc_wfn.set_variable(f"{name} state dipoles", core.Matrix.from_array(data))
+    computed = []
+    method = name.upper()
```

Comment:
@loriab can correct me if I'm wrong, but I think we want the variables set with the name as both "ADC" and, for example, "ADC(2)". This part of the specification only came out during some of my other PRs.

Reply:
you're right. definitely the method (e.g., "ADC(2)"). there's isn't a great overarching label, though. "SCF", "[TD-]DFT", "CC", "CI" are the ones in use, I think. I've just checked detci and cc* for any MPBT or MPN variants. So unless you can think of a likely intermediate between "ADC(2) TOTAL ENERGY" and "CURRENT ENERGY", I'm happy with just the method qcvar.

Reply:
Except I proposed such a likely intermediate, "ADC property-name-here"?

Reply:
true. not quite so broad as "CI", imo. I'm good either with or without "ADC". maybe mildly "for" since those vars exist: https://psicode.org/psi4manual/master/autodir_psivariables/module__adc.html

Reply:
Okay, I will add the generic "ADC" name 👍